test/ExpressionSplitter.test.ts

```diff
@@ -0,0 +1,20 @@
+import { splitExpression } from '../src/ExpressionSplitter';
+
+describe('splitExpression', () => {
+	test('should handle escaping backslashes before double opening curly braces', () => {
+		const expr = 'C:\\\\Users\\\\Administrator\\\\Desktop\\\\abc\\\\{{ $json.files[0].fileName }}';
+		const result = splitExpression(expr);
+
+		expect(result).toEqual([
+			{
+				type: 'text',
+				text: 'C:\\Users\\Administrator\\Desktop\\abc\\',
+			},
+			{
+				type: 'code',
+				text: ' $json.files[0].fileName ',
+				hasClosingBrackets: true,
+			},
+		]);
+	});
+});
```

Comment:
Other tests already exist for `splitExpression` in https://github.com/n8n-io/tournament/blob/main/test/ExpressionExtension.test.ts

Do we want to move this test in there and perhaps rename the module to `ExpressionSplitter.test.ts` for consistency and clarity?